subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -639,7 +658,8 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
 	aux->ull.ticks_slot =
 		HAL_TICKER_US_TO_TICKS(EVENT_OVERHEAD_START_US +
 				       ready_delay_us +
-				       PDU_AC_MAX_US(0U, lll_aux->phy) +
+				       PDU_AC_MAX_US(PDU_AC_EXT_PAYLOAD_SIZE_MAX,
```

Comment:
It looks that this change reverts change done in this PR: 779d523c487a55267113177b617ffef79df90963.

Reply:
Yes, the use of ticker_yield_abs does not need minimal reservation. I would like to know the commit with minimal reservation for history of the issue being addressed.